spec/acceptance/ios_aaa_authentication_spec.rb

```diff
@@ -48,4 +78,56 @@
     result = run_resource('ios_aaa_authentication', "'ppp default'")
     expect(result).to match(%r{ensure.*absent})
   end
+
+  it 'apply aaa authentication onep' do
```

Comment:
is there something special about onep? I would suggest that a more worthy test would be with `suppress` and `suppress_null_username` combos.